protect-api/src/main/java/com/contoso/api/ApiController.java

```diff
@@ -0,0 +1,21 @@
+package com.contoso.api;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/api")
+public class ApiController {
+
+    // This annotation will trigger MSAL for Java to validate the JWT provided
+    // and also will require that it contains a scope claim of "Greeting.Read"
+    // An invalid or missing JWT token will result in a 401, a token without
+    // sufficent scope will result in a 403.
+    @GetMapping(value = "/hello", produces = "text/plain")
+    @PreAuthorize("hasAuthority('SCOPE_Greeting.Read')")

```

Comment:
this looks like a bit different for other scopes we were using in the past like `Forecast.Read`.

Reply:
`Greeting.Read` is different than `Forecast.Read` how?  I mean, besides the name.  This endpoint doesn't return a forcast.

Reply:
If your question is about the `SCOPE_` prefix, that's a Spring Boot + Azure AD package implementation detail (read: a teachable component for the tutorial :))

Reply:
yeah my point was around the `SCOPE_` prefix. Am I understanding this correctly if I say that this is required by convention when suing Spring Boot?

Reply:
Correct, it's a Spring Boot + Azure AD (MSAL) Security handler convention.  The `SCOPE_` part is not in the actual name of the scope in Azure AD.

Reply:
it was just a comment in here after approval, so please let's consider this resolved.